websocket/_examples/sendreceive/main.go

```diff
@@ -0,0 +1,70 @@
+package main
```

Comment:
I would've split this file in two separate binaries:
- One for a server
- One for a client

An user can experiment with these binaries by creating a server and multiple clients to connect to the server

Reply:
I prefer as it is